### DISCUSSION TITLE
Sistema Antimulta

### DISCUSSION THREAD

**Discussion by vish-blog** [vish-blog/comments#408] [Imagens]
Post de referência: https://www.vish.blog.br/imagens/sistema-antimulta/

**Comment by one-eyed-willy** [comment]
Eu não dirijo nem carrinho de bate-bate, mas R$ 7,24? :O

**Comment by angellisy** [comment]
> [Willy Caolho](</perfil/?user=one-eyed-willy>) disse [sábado, 30 de abril de 2022](<https://www.vish.blog.br/imagens/sistema-antimulta/#comment_1114003462>):

> Eu não dirijo nem carrinho de bate-bate, mas R$ 7,24? :O

Achou que Brasil tava brincando?
